style.css

```diff
@@ -603,6 +603,196 @@ h2 {
   width: 25rem;
 }
 
+/* modal */
+.modal-window {
+  z-index: 5;
+  position: fixed;
+  top: 10%;
+  left: 5%;
+  width: 90%;
+  margin: 0 auto;
+  background-color: #fff;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  box-sizing: border-box;
+  padding: 10px;
+  max-height: 554px;
+  overflow-y: scroll;
+}
```

Comment:
- [ ] Kindly give more height as in the design to your pop-up window. You should remove line 621 as it gives a short height and make it scrollable. Right now, your popup seems like this: 
![image](https://user-images.githubusercontent.com/91486791/195604013-532ad2ad-6cc1-49b2-b076-49b367f9d6d0.png)
![image](https://user-images.githubusercontent.com/91486791/195604275-bf6a7fd1-41ca-4319-a9eb-4fb9eb10b88a.png)
on mobile screens: 
![image](https://user-images.githubusercontent.com/91486791/195604422-810202ee-7b7c-40f7-b226-b4726fe763f0.png)

figma design: 
![image](https://user-images.githubusercontent.com/91486791/195604599-1b8ada7f-e211-4c81-b538-f7722dc30cca.png)
![image](https://user-images.githubusercontent.com/91486791/195604701-3a4f6d5b-5a09-4da1-8074-47aee03303ca.png)